crates/ingress-kafka/src/subscription_controller.rs

```diff
@@ -140,6 +140,7 @@ impl Service {
         // see ConsumerTask::run
         client_config.set("enable.auto.commit", "true");
         client_config.set("enable.auto.offset.store", "false");
+        client_config.set("statistics.interval.ms", "10000");
```

Comment:
I think enabling this should be opt-in, and not hardcoded.

If the user sets the `statistics.interval.ms` in the cluster options/subscription metadata, then statistics are enabled.